source/features/wait-for-attachments.ts

```diff
@@ -0,0 +1,50 @@
+// GitHub natively marks `.js-upload-markdown-image textarea.js-comment-field` as invalid while a file is uploading.
+// `textarea[data-required-trimmed]` conflicts with this behavior by overriding the validity states in the meantime.
+// `button[data-disable-invalid]` are automatically disabled while the form is invalid, but some buttons don't have it.
+
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+const attribute = 'data-required-trimmed';
+const attributeBackup = 'data-rgh-required-trimmed';
+
+function toggleSubmitButtons({target, type}: Event): void {
+	const fileAttachment = target as HTMLElement;
+
+	// Ensures every submit button has the `data-disable-invalid` attribute
```

Comment:
Doesn't this affect the "Close and comment" button? That should never be disabled. I think the selector needs to be more specific to the buttons we explicitly support, or else we risk breaking submission in these edge cases

Reply:
Yes, the "Close with comment" is affected since it's a submit button.

> the buttons we explicitly support

What are those exactly? Just the green ones?

Reply:
Apparently: 

https://github.com/sindresorhus/refined-github/pull/3886/commits/4837311169006bdf54131e2c4a5f6c6887add4fc#diff-de5c204d3ff1280d1b01a540e2b88a77fc7441a87265233b04915a1e9c2eccb0L29

Therefore it's just the "Update comment", that we know of, at this time.

Reply:
I changed the selector to target only primary submit buttons. What about the "Create draft PR" button? Should we leave it unaffected too?

Reply:
It could be ok for now